### PR TITLE
Add phpstan/psalm

### PR DESCRIPTION
**Pull request by VincentLanglet**
Pedantic

I'll do the PR on dev-kit after.

### CODE REVIEW

File: psalm.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <directory name="tests"/>
+        <ignoreFiles>
+            <directory name="vendor"/>
+        </ignoreFiles>
+    </projectFiles>
+    <plugins>
+        <pluginClass class="Psalm\SymfonyPsalmPlugin\Plugin"/>
+        <pluginClass class="Psalm\PhpUnitPlugin\Plugin"/>
+    </plugins>
+    <issueHandlers>
```
Comment (jordisala1991) [inline psalm.xml:14]: this should not be here, like we did on media-bundle. Probably that should be changed on admin-bundle too

**Comment by VincentLanglet** [comment]
Created https://github.com/phpstan/phpstan/issues/5590 for the tests failure

**Comment by jordisala1991** [comment]
If Fosuser bundle have fixed the issues with doctrine persistence we can bump min version and remove those lines

**Comment by VincentLanglet** [comment]
> If Fosuser bundle have fixed the issues with doctrine persistence we can bump min version and remove those lines

Fixed

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**